Library/Homebrew/upgrade.rb

```diff
@@ -375,6 +375,10 @@ def self.upgrade_dependents(deps, formulae,
         puts formulae_upgrades.join(", ")
       end
 
+      upgradeable = upgradeable.reject { |f| FormulaInstaller.installed.to_a.include?(f) }
```

Comment:
```suggestion
      upgradeable.reject! { |f| FormulaInstaller.installed.include?(f) }
```
Isn't this equivalent? `FormulaInstaller.installed` is a `Set`, so it should already support `#include?`.